csrc/angle/harmonic_angle_cuda.cu

```diff
@@ -146,7 +142,7 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> compute_harmonic_angle_energy_gra
 ) {
     int64_t nangles = triplets.size(0);
 
-    int block_dim = 1024;
+    int block_dim = 512;
```

Comment:
Somehow I found the cuda cannot maintain `blockDim = 1024` when double precision is used. I assume it's because there will not have that enough register allocated for each thread. `blockDim = 512` will work. There are be some ways to rewrite the cuda kernel to not using too many registers, but we can leave it now I think.